src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptor.cs

```diff
@@ -28,6 +46,7 @@ public class TagHelperDescriptor
         /// Instantiates a new instance of the <see cref="TagHelperDescriptor"/> class with the given
         /// <paramref name="attributes"/>.
         /// </summary>
+        /// <param name="prefix">The prefix of the HTML element name the tag helper targets.</param>
```

Comment:
I'd much rather not add complexity to the `TagHelperDescriptor`, it should represent everything that makes up a valid `TagHelper`, in this case the HTMLs `TagName` (what should eventually hit the page) and its optional `Prefix`. Not to mention the editor uses these `TagHelperDescriptor`s and will also use the `Prefix` property directly.